### PR TITLE
UPC-49 End of support of plugin parent relation

### PR DESCRIPTION
**Pull request by simonbrandhof**
UPC-49 End of support of plugin parent relation

### CODE REVIEW

File: sonar-update-center-common/src/test/java/org/sonar/updatecenter/common/PluginManifestTest.java [hunk 1]
```diff
@@ -90,15 +88,6 @@ public void should_add_sources_url() throws URISyntaxException, IOException {
   }
 
   @Test
-  public void should_add_parent() throws URISyntaxException, IOException {
-    URL jar = getClass().getResource("/org/sonar/updatecenter/common/PluginManifestTest/plugin-with-parent.jar");
```
Comment (henryju) [inline sonar-update-center-common/src/test/java/org/sonar/updatecenter/common/PluginManifestTest.java]: Don't forget to remove resource plugin-with-parent.jar if it is no more used.

**Comment by henryju** [comment]
Except cleanup of the test resource LGTM

**Comment by sns-seb** [comment]
I don't see any change in Ruby code even though references are made to children list of `Release`, for example, in `server/sonar-web/src/main/webapp/WEB-INF/app/views/updatecenter/available.html.erb` ligne 54.

**Comment by sns-seb** [comment]
ok, got it, this is done in PR #226

it's confusing to have related commits in separates PRs nut I guess it's make review easier...

File: sonar-update-center-common/src/test/java/org/sonar/updatecenter/common/PluginManifestTest.java (hunk 1 above)
Comment (simonbrandhof) [inline sonar-update-center-common/src/test/java/org/sonar/updatecenter/common/PluginManifestTest.java]: good catch

**Pull request closed without merging**